storagezone_add.go

```diff
@@ -0,0 +1,38 @@
+package bunny
+
+import "context"
+
+// StorageZoneAddOptions are the request parameters for the Get Storage Zone API endpoint.
+//
+// Bunny.net API docs: https://docs.bunny.net/reference/storagezonepublic_add
+type StorageZoneAddOptions struct {
+	// The origin URL of the storage zone where the files are fetched from.
+	OriginURL string `json:"OriginUrl,omitempty"`
+	// The ID of the storage zone that the storage zone is linked to.
+	Region string `json:"Region,omitempty"`
+
+	// The name of the storage zone (Optional)
+	Name string `json:"Name,omitempty"`
+	// The code of the main storage zone region (Optional)
+	ReplicationRegions string `json:"ReplicationRegions,omitempty"`
```

Comment:
Please use pointers for those fields.
The bunny api does not define if it always assumes the same values then Go if fields are omitted.
By using pointers we can distinguish clearly between unset and default values. This prevents discrepancy when interpreting the unset fields.